.gitignore

```diff
@@ -1,4 +1,5 @@
 zig-cache/
+.zig-cache/
```

Comment:
I'm not sure if this is just on my system or if it is zls that is doing it. But these folders show up for me.